.jenkins/pytorch/win-test.sh

```diff
@@ -59,3 +64,16 @@ run_tests() {
 }
 
 run_tests && assert_git_not_dirty && echo "TEST PASSED"
+
+if [[ "${BUILD_ENVIRONMENT}" == "pytorch-win-vs2019-cuda10-cudnn7-py3" ]]; then
```

Comment:
Can this be enabled only for test1? (to save some CPU cycles while running test2)
```suggestion
if [[ "${BUILD_ENVIRONMENT}" == "pytorch-win-vs2019-cuda10-cudnn7-py3"  && "${JOB_BASE_NAME}" == *-test1]]; then
```

Reply:
Wouldn't this change lower the coverage? Not all tests run in test1.

Reply:
No, because we are still getting coverage from linux cpu config, which are cheaper and faster to run and codecov should merge coverage from multiple reports.

Reply:
@guyang3532 Could you please try disabing this? If the coverage is okay, I'm fine with it.

Reply:
done